solution/text-extractor/backends/s3.py

```diff
@@ -0,0 +1,35 @@
+import boto3
+import botocore.exceptions
+
+from .backend import FileBackend
+from .exceptions import BackendException, BackendInitException
+
+
+class S3Backend(FileBackend):
+    backend = "s3"
```

Comment:
I could be wrong here but I think the point of setting the backend is so that on post_params you can easily change it.  

So it would be post_params[self.backend]["aws_access_key_id"].  That way if someone changes the class they can just e like x = S2Backend()
x.backend= 'whatever'
then the post params are more dynamic